srcpkgs/rdma-core/template

```diff
@@ -0,0 +1,12 @@
+# Template file for 'rdma-core'
+pkgname=rdma-core
+version=22.1
+revision=1
+build_style=cmake
+hostmakedepends="cmake gcc libnl3-devel eudev-libudev-devel make pkg-config python3-devel valgrind-devel"
+short_desc="RDMA core userspace libraries and daemons"
+maintainer="Rich G <rich@richgannon.net>"
+license="GPL-3.0-or-later"
```

Comment:
Wrong license, please check upstream for correct license

Reply:
Corrected in new commit.